applications/serial_lte_modem/overlay-native_tls.conf

```diff
@@ -57,9 +59,11 @@ CONFIG_MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED=y
 CONFIG_MBEDTLS_KEY_EXCHANGE_ECDHE_RSA_ENABLED=y
 # Enable RSA
 CONFIG_MBEDTLS_KEY_EXCHANGE_RSA_ENABLED=y
-CONFIG_MBEDTLS_RSA_C=y
```

Comment:
@mswarowsky: Apologies, this is still needed. Blaming this one to lack of sleep. :)

Reply:
No worries, thanks for correcting